core-bundle/src/Resources/contao/classes/BackendTemplate.php

```diff
@@ -74,6 +79,9 @@ protected function compile()
 			$this->ua .= ' fullscreen';
 		}
 
+		// Add backend config to the template @see #403
```

Comment:
I don't think that comment makes a lot of sense :)

Reply:
Oh copy & paste from previous draft - yesterday I even thought about removing this comment :-D